lib/instruments.js

```diff
@@ -149,8 +150,8 @@ class Instruments {
     this.proc.stderr.setEncoding('utf8');
     let actOnStderr = (output) => {
       if (this.launchTimeout.afterSimLaunch && output && output.match(/CLTilesManagerClient: initialize/)) {
-        this.addSocketConnectTimer(this.launchTimeout.afterSimLaunch, 'afterLaunch', async () => {
-          await killAllInstruments();
+        this.addSocketConnectTimer(this.launchTimeout.afterSimLaunch, 'afterLaunch', async (done) => {
```

Comment:
Do we not get the error message and rejection done even on success, then? It looks like `done` is called all over the place in `killInstruments`.